src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml

```diff
@@ -30,9 +34,34 @@
                 </Style.Triggers>
             </Style>
 
+            <Style x:Key="DeleteButtonStyle" TargetType="Button" BasedOn="{StaticResource ToolWindowButtonStyle}">
+                <Style.Triggers>
+                    <Trigger Property="IsEnabled" Value="True">
+                        <Setter Property="ToolTip" Value="{x:Static components:Resources.ConnectionManager_DeleteTooltip}" />
+                    </Trigger>
+                    <Trigger Property="IsEnabled" Value="False">
+                        <Setter Property="ToolTip" Value="{x:Static components:Resources.ConnectionManager_DeleteTooltipDisabled}" />
+                    </Trigger>
+                </Style.Triggers>
+                <Setter Property="Opacity" Value="{Binding Path=IsUserCreated, Converter={x:Static rwpf:Converters.FalseIsGrayedOut}}" />
+                <Setter Property="IsEnabled" Value="{Binding Path=IsUserCreated}" />
+                <Setter Property="ToolTipService.ShowOnDisabled" Value="True" />
+            </Style>
+
+            <Style x:Key="EditButtonStyle" TargetType="Button" BasedOn="{StaticResource ToolWindowButtonStyle}">
+                <Style.Triggers>
+                    <DataTrigger Binding="{Binding Path=IsRemote}" Value="False">
+                        <Setter Property="ToolTip" Value="{x:Static components:Resources.ConnectionManager_EditLocalTooltip}" />
+                    </DataTrigger>
+                    <DataTrigger Binding="{Binding Path=IsRemote}" Value="True">
+                        <Setter Property="ToolTip" Value="{x:Static components:Resources.ConnectionManager_EditRemoteTooltip}" />
+                    </DataTrigger>
+                </Style.Triggers>
+            </Style>
+
             <DataTemplate x:Key="EditConnectionDataTemplate" DataType="{x:Type viewModel:IConnectionViewModel}">
                 <AdornerDecorator>
-                    <Grid>
+                    <Grid Margin="0,6,0,8">
```

Comment:
When editing it better separates the editing block from connection entries above and below.